packages/core/src/components/Checkbox/types.ts

```diff
@@ -11,6 +11,7 @@ export interface ICheckboxProps extends CommonProps {
   isIndeterminate?: boolean;
   name?: string;
   isRequired?: boolean;
+  ref?: any;
```

Comment:
Should we use `HTMLInputElement` instead of `any` here?

Reply:
Thanks for sharing your idea. I am going to use `MutableRefObject<HTMLInputElement>` here